policy/datasets.yaml

```diff
@@ -99,6 +100,7 @@ deepCoadd_ref_schema:
     python: lsst.afw.table.SourceCatalog
     template: schema/deepCoadd_ref.fits
 deepCoaddId:
+    description: "A unique identifier for a coadd of a single patch. This is overridden by the CameraMapper and not a persisted dataset."
```

Comment:
Can you line break this please? It's easier to read if the lines are shorter (especially on GitHub)